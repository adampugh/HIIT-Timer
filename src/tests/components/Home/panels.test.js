import React from "react";
import { shallow } from "enzyme"; 

import Panels from "../../../components/Home/panels";

test("should render panels correctly", () => {
    const wrapper = shallow(<Panels />);
    expect(wrapper).toMatchSnapshot();
});