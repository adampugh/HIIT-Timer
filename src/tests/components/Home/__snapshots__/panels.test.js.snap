// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render panels correctly 1`] = `
<div
  className="homecontent__wrapper"
>
  <div
    className="container homecontent"
  >
    <h2>
      Create workouts today and get in shape!
    </h2>
    <div
      className="grid--3to1 homecontent__grid"
    >
      <div
        className="homecontent__panel"
      >
        <img
          alt="Arm icon"
          src="arm-icon.png"
        />
        <h3>
          Create Workouts
        </h3>
        <p>
          Create workouts from scratch by first entering a workout title. Add exercises and customize the workout. Start the clock and get exercising!
        </p>
      </div>
      <div
        className="homecontent__panel"
      >
        <img
          alt="Dumbbell icon"
          src="dumbbell-icon.png"
        />
        <h3>
          Customize Workouts
        </h3>
        <p>
          Add exercises that suit your goals for losing weight or building muscle. Choose to include breaks and different colours for each exercise.
        </p>
      </div>
      <div
        className="homecontent__panel"
      >
        <img
          alt="Pencil icon"
          src="pencil-icon.png"
        />
        <h3>
          Edit Workout
        </h3>
        <p>
          Delete and add workouts to adapt to your schedule or workout regime. Update exercises to continue your progression and help you reach your goals.
        </p>
      </div>
    </div>
  </div>
</div>
`;
